### PR TITLE
add back notifications

### PR DESCRIPTION
**Pull request by aeddaqqa**
add back notifications to the wallet using an action to be dispatched in the suite when the notification is triggered

### CODE REVIEW

File: src/components/misc/CopyText.vue [hunk 1]
```diff
@@ -18,7 +18,8 @@ export default {
             let copytext = this.$refs.copytext
             copytext.select()
             copytext.setSelectionRange(0, 99999)
-
+            let { dispatchNotification } = this.globalHelper()
+            dispatchNotification({ message: 'Copied to clipboard', type: 'success' })
```
Comment (Chriserus) [inline src/components/misc/CopyText.vue]: Please use translation keys in wallet

**Review by Chriserus — DISMISSED** [review]

File: src/components/misc/CopyText.vue (hunk 1 above)
Comment (Ayagoumi) [reply src/components/misc/CopyText.vue]: @aeddaqqa And also unify the wording letter case (Upper and lower case).

File: src/components/wallet/earn/ChainTransfer/ChainTransfer.vue [hunk 2]
```diff
@@ -533,7 +538,11 @@ export default class ChainTransfer extends Vue {
     onsuccess() {
         // Clear Form
         this.isSuccess = true
-
+        let { dispatchNotification } = this.globalHelper()
+        dispatchNotification({
+            message: 'Funds transferred between chains.',
+            type: 'error',
```
Comment (Ayagoumi) [inline src/components/wallet/earn/ChainTransfer/ChainTransfer.vue]: This should be success

**Review by Chriserus — DISMISSED** [review]

File: src/components/wallet/earn/ChainTransfer/ChainTransfer.vue (hunk 2 above)
Comment (aeddaqqa) [reply src/components/wallet/earn/ChainTransfer/ChainTransfer.vue]: fixed

File: src/components/misc/CopyText.vue (hunk 1 above)
Comment (aeddaqqa) [reply src/components/misc/CopyText.vue]: done

**Review by Ayagoumi — APPROVED** [review]

**Pull request merged**